.ci/pytorch/test.sh

```diff
@@ -335,12 +335,23 @@ test_inductor_benchmark_perf() {
   # Not checking accuracy for perf test for now
   # shellcheck disable=SC2086
   if [[ "$1" == *smoketest* ]]; then
-    python benchmarks/dynamo/torchbench.py --performance --backend inductor --float16 --training \
+    python benchmarks/dynamo/torchbench.py --device cuda --performance --backend inductor --amp --training \
```

Comment:
@ngimel , I realized your suggested perf smoke test was using `--float16` instead of `--amp`. Is there any particular reason for that?

Reply:
No particular reason, I think `--float16` is a bit more stable and isolates inductor part better, but I don't have hard data to back that up.